Configuration/TCA/tx_dlf_metadataformat.php

```diff
@@ -25,9 +25,6 @@
         'searchFields' => 'encoded',
         'hideTable' => 1,
     ],
-    'feInterface' => [
-        'fe_admin_fieldList' => '',
-    ],
     'interface' => [
     ],
     'columns' => [
```

Comment:
When trying to add a metadata format, I got an error that l18n_diffsource does not have a default value. Should we add the `ctrl.transOrigDiffSourceField` and `columns.l18n_diffsource` fields like for the other tables?

Reply:
How to you add an metadata format record? I add it as inline record inside the metadata record. I get no error.
Did you made a database compare? I added the language field for `tx_dlf_metadataformats` again, because I had similar problems before.

Reply:
Okay, the issue has been that my database still had the l18n_diffsource column even though it had been removed from ext_tables. Did you want to also bring back that column? Otherwise, the fix would be to remove the column via "Maintenance → Analyze Database Structure".

Reply:
No, this column is not needed and was (IMHO) introduced during development. And it's totally ok for a major release to require a full database compare.